sites/rdhmag/server/templates/index.marko

```diff
@@ -229,7 +162,7 @@ $ const { id, alias, name, pageNode } = data;
         component-input={ aliases }
         fragment-name="content-list"
         query-name="website-scheduled-content"
-        query-params={ sectionId: id, limit: 14, skip: 5 }
+        query-params={ sectionId: id, limit: 10, skip: 12 }
```

Comment:
@Alixers505 - Shouldn't this skip remain at 5?

Reply:
I updated it to 10/12 because I saw that's what the current site is utilizing.

Reply:
@Alixers505 - limit 10 is correct, since they only show the card deck in load more (they don't sell 300x600s).  The home page skip is actually wrong on their Live site; it should be 5 since the Hero block is the only block set to query the home section.  If they had a "More from RDH" block, like some sites do, then this limit would be a little higher.